src/lib/pages/workflow-history-json.svelte

```diff
@@ -1,6 +1,22 @@
 <script lang="ts">
+  import { page } from '$app/stores';
+
   import CodeBlock from '$lib/components/code-block.svelte';
-  import { events } from '$lib/stores/events';
+  import Loading from '$lib/components/loading.svelte';
+  import { fetchRawEvents } from '$lib/services/events-service';
+  import { decodeURIForSvelte } from '$lib/utilities/encode-uri';
+
+  const { namespace, workflow: workflowId, run: runId } = $page.params;
+  const events = fetchRawEvents({
+    namespace: decodeURIForSvelte(namespace),
```

Comment:
It's kind of gross that we need to do this outside of the service itself. Why doesn't `events-service.ts` just handle this under the hood?